### DISCUSSION TITLE
vc-graphics-ellipsoid组件动态改变maximum-clock属性会报错

### DISCUSSION THREAD

**Discussion by mc15829702687** [zouyaoji/vue-cesium#294] [Q&A]
vc-graphics-ellipsoid组件，在动态改变maximum-clock属性时会渲染停止
![f2f6553a0c87d39e6acfd6248d9df3c](https://user-images.githubusercontent.com/44308726/196362618-df80161c-4563-44c8-8d30-58a15b52bcc9.jpg)
![e3fd95fad6905058ad49b370b050e19](https://user-images.githubusercontent.com/44308726/196363029-49db8915-9131-4795-aa66-6ac393d0f224.jpg)

**Comment by zouyaoji** [comment]
你咋写的~

_cesium.Math 大概率取不到吧

**Reply by mc15829702687** [reply]
大佬，可以取到，我把Cesium放到ref里了，然后名字是_cesium

**Reply by zouyaoji** [reply]
没能重现哈
![录制_2022_10_19_01_11_41_757](https://user-images.githubusercontent.com/24430116/196499305-8cd6b001-e51b-4c14-aff9-fee1b1aee502.gif)

**Reply by mc15829702687** [reply]
麻烦大佬了，应该是我的问题，我再看看，十分感谢

**Reply by zouyaoji** [reply]
https://github.com/zouyaoji/vue-cesium/issues/155#issuecomment-1042470701

**Reply by mc15829702687** [reply]
谢谢大佬，应该不是识别Cesium对象的问题，我再看看是不是我代码哪有问题
